Files/Interacts/Interaction.cs

```diff
@@ -58,6 +58,28 @@ public void List_ItemClick(object sender, DoubleTappedRoutedEventArgs e)
             OpenSelectedItems(false);
         }
 
+        public async void List_ItemPress(object sender, PointerRoutedEventArgs e)
+        {
+            if (e.Pointer.PointerDeviceType == Windows.Devices.Input.PointerDeviceType.Mouse)
+            {
```

Comment:
We'd want to have this condition handle touch input instead of just Mouse.

Reply:
Thank you for the feedback, this should be resolved.